kubernetes/linux/setup.sh

```diff
@@ -44,8 +44,8 @@ sudo apt-get update
 sudo apt-get install td-agent-bit=1.6.8 -y
 
 # install ruby2.6
-sudo apt-get install software-properties-common -y
-sudo apt-add-repository ppa:brightbox/ruby-ng -y
+sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys F5DA5F09C3173AA6
```

Comment:
looks ok to me. Only question is if we can get the keys from keyserver (jusu like we do for fluentbit).

Reply:
Yesterday, I have spent good amount of time to figure out if we can use the key file like we did FBIT but no luck on that. Hence, the proceed with approaching of getting key from ubuntu key server and which is other good alternate option.  AFIK, this seems better available option but if you have way to locate the key file and i am happy to change to that.